tests/test_modeling_gpt2.py

```diff
@@ -173,7 +175,7 @@ def create_and_check_lm_head_model(self, config, input_ids, input_mask, head_mas
 
             self.parent.assertListEqual(list(result["loss"].size()), [])
             self.parent.assertListEqual(
-                list(result["lm_logits"].size()), [self.batch_size, self.seq_length, self.vocab_size]
+                list(result["lm_logits"].size()), [self.batch_size, self.seq_length, self.vocab_size],
```

Comment:
(nit) you can also do 
`self.parent.assertEqual(x.size(), (bs, seq_len, vocab_size)`
Which I would prefer but no strong feelings.